src/index.ts

```diff
@@ -44,11 +44,12 @@ export default {
         incident.updated_at = new Date().toISOString();
       }
 
-      if (kv === null) {
-        await this.postNew(incident, env);
-      } else {
-        await this.postUpdate(incident, kv, env);
-      }
+      // If we've not seen this before, and it's already resolved, ignore it
+      // This stops us spamming old incidents on first run
+      if (kv === null && incident.status === 'resolved') return;
```

Comment:
My only hesitation here is that this would break the detection of "retro" incidents that're posted in a resolved status from the start.

I could get around this by storing a new KV value that tracks if this is the first run, and writing these "skipped" incidents to KV with a different data format that indicates they were skipped.

Reply:
Maybe we should [`KV.list`](https://developers.cloudflare.com/workers/runtime-apis/kv/#listing-keys) and if size is 0 then put all incidents into KV but don't post.

Reply:
`KV.list` has a limit of 1,000 a day before being billed, and with this running every minute via cron, that'd mean 440 billed list calls each day. I'll go down the route of just storing a `firstRun` key in KV and check if that exists.